docs/rkt/README.md

```diff
@@ -0,0 +1,34 @@
+# Integrating the Calico Network Plugin with rkt
+
+This guide will describe the configuration required to use the Calico network plugin in your rkt deployment.
+
+## Using rkt Plugins
+
+The CoreOS [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins will walk you through the basics of setting up networking in rkt.
+
+## Requirements
+
+* A working [etcd](https://github.com/coreos/etcd) service
+* A build of `calicoctl` after [projectcalico/calico-docker@10460cc405](https://github.com/projectcalico/calico-docker/commit/10460cc405f5aa4bc9ccb1fcaf8760088ae1ebf9)
+* Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a [docker container](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md). While we work on native rkt support, you will need to run Calico in Docker before starting your rkt containers. This can be easily done wtih `calicoctl` by running the following command: `sudo calicoctl node --ip=<IP> --rkt`
+
+## Installing
+
+* Running `calicoctl node` with the `--rkt` flag will start the calico/node process and automatically install the plugin for you. Alternatively you can download the [plugin binary](https://github.com/projectcalico/calico-rkt/releases/) yourself and move it to the rkt plugin directory.
+```
+chmod +x calico_rkt
+sudo mv -f ./calico_rkt /usr/lib/rkt/plugins/net/calico
+```
+
+## Building the plugin locally
+
+To build the Calico Networking Plugin for rkt locally, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
```

Comment:
"clone this repository and run `make`" -- This is no longer correct, since we've moved to a new repo.

Generally, we've kept development, building, and testing instructions within the README for each repo, and only moved installation instructions, guides, and more detailed documentation to calico-docker.